docker/entrypoints/webpack.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+rm -rf /app/tmp/pids/server.pid
+rm -rf /app/tmp/cache/webpacker/*
+
+echo "Waiting for yarn and bundle integrity to match lockfiles...."
+# YARN="yarn check --integrity"
```

Comment:
why did we comment this out?

Reply:
For some reason the docker integrity checks were not working and @petebytes commented out webpack yarn integrity check and i am assuming it was due to the docker volumes acting weird with integrity check working in `webpack service` and not in `rails service`. I have tweaked the scripts to make the yarn integrity work. It would be good if someone could test it other than in my machine.